pkg/apis/core/types.go

```diff
@@ -1634,6 +1634,10 @@ type CSIPersistentVolumeSource struct {
 	// Ex. "ext4", "xfs", "ntfs". Implicitly inferred to be "ext4" if unspecified.
 	// +optional
 	FSType string
+
+	// Attributes of the volume to publish.
+	// +optional
+	VolumeAttributes map[string]string
```

Comment:
why does this want the "Volume" prefix?  Does it feel redundant in actual use?

Reply:
It's to align with the parameter `volume_attributes` in the CSI spec.